frontend/src/styles/modules/_modals.scss

```diff
@@ -318,6 +318,98 @@
   .button {
     margin: 0 auto;
   }
+
+  &.news-updates-modal {
```

Comment:
nit: Would like to see this in a `frontend/src/app/components/NewsUpdatesDialog/index.scss` file. But, #2808 & #2889 are probably better issues to address this, since the shared modal styles are kind of tangled together